kDriveCore/Data/Models/Rights.swift

```diff
@@ -73,6 +75,7 @@ public class Rights: EmbeddedObject, Codable {
         case canUpload
         case canMoveInto
         case canBecomeDropbox
+        case canColor
```

Comment:
canColor key doesn't exist in the json
```suggestion
        case canColor = "colorable"
```